src/bundle/Controller/SearchController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\Controller;
+
+use eZ\Publish\Core\Repository\SearchService;
+use EzSystems\EzPlatformAdminUi\Form\Data\Search\SearchData;
+use EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper;
+use Pagerfanta\View\TwitterBootstrap4View;
+use Symfony\Component\HttpFoundation\Request;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use Pagerfanta\Pagerfanta;
+use eZ\Publish\Core\Pagination\Pagerfanta\ContentSearchAdapter;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
+use EzSystems\EzPlatformAdminUi\Form\SubmitHandler;
+
+class SearchController extends Controller
+{
+    /** @var SearchService */
+    private $searchService;
```

Comment:
I would define all properties either `private` or `protected`. `private` seems to be a more popular choice with recent PRs in order to disallow relying someone's code on our controllers.